specifyweb/frontend/js_src/lib/components/FormSliders/RecordSelectorFromIds.tsx

```diff
@@ -239,6 +239,7 @@ export function RecordSelectorFromIds<SCHEMA extends AnySchema>({
             : undefined
         }
         onSaved={(): void => handleSaved(resource!)}
+        classNameProp="w-[inherit]"
```

Comment:
rename to `className` here to match how such prop is called in other places
internally in ResourceView, you can rename it to avoid conflict:
```
className: classNameProp
```

Reply:
is this class name added here to prevent the dialog from remembering size?
not sure how that fixes the issue, and also not sure why it was added just in this case, rather than in all the cases where ResourceView displays as a dialog

Reply:
<<rather than in all the cases where ResourceView displays as a dialog>> 
Because I didn't want it to be apply to the ResourceView 'Create Record Set' by example. Here I can apply it only for the ones I need the w-inherit